Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-alpine
+WORKDIR /code
+RUN apk add --no-cache gcc musl-dev linux-headers
+RUN apk update && apk add libpq
+RUN apk add --virtual .build-deps gcc python-dev musl-dev postgresql-dev
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+COPY . .
```

Comment:
Why do we need both `COPY .` and `COPY requirements.txt` commands here?

Reply:
My fail, didn't COPY requirements.txt